.github/workflows/prod-ci-cd.yml

```
@@ -0,0 +1,99 @@
+name: Production CI/CD
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)